dbt/seeds/etl_full_row_counts.csv

```diff
@@ -3143,37 +3142,6 @@ out_ferc1__yearly_purchased_power_and_exchanges_sched326,2021,6899
 out_ferc1__yearly_purchased_power_and_exchanges_sched326,2022,7052
 out_ferc1__yearly_purchased_power_and_exchanges_sched326,2023,7066
 out_ferc1__yearly_purchased_power_and_exchanges_sched326,2024,7065
-out_ferc1__yearly_rate_base,1994,14718
-out_ferc1__yearly_rate_base,1995,14407
-out_ferc1__yearly_rate_base,1996,14810
-out_ferc1__yearly_rate_base,1997,14547
-out_ferc1__yearly_rate_base,1998,14309
-out_ferc1__yearly_rate_base,1999,14340
-out_ferc1__yearly_rate_base,2000,14450
-out_ferc1__yearly_rate_base,2001,14032
-out_ferc1__yearly_rate_base,2002,13434
-out_ferc1__yearly_rate_base,2003,13600
-out_ferc1__yearly_rate_base,2004,12667
-out_ferc1__yearly_rate_base,2005,14043
-out_ferc1__yearly_rate_base,2006,14093
-out_ferc1__yearly_rate_base,2007,13818
-out_ferc1__yearly_rate_base,2008,13907
-out_ferc1__yearly_rate_base,2009,13634
-out_ferc1__yearly_rate_base,2010,13780
-out_ferc1__yearly_rate_base,2011,13511
-out_ferc1__yearly_rate_base,2012,13621
-out_ferc1__yearly_rate_base,2013,13289
-out_ferc1__yearly_rate_base,2014,13270
-out_ferc1__yearly_rate_base,2015,13145
-out_ferc1__yearly_rate_base,2016,13212
-out_ferc1__yearly_rate_base,2017,13311
-out_ferc1__yearly_rate_base,2018,13367
-out_ferc1__yearly_rate_base,2019,13597
-out_ferc1__yearly_rate_base,2020,13703
-out_ferc1__yearly_rate_base,2021,15219
-out_ferc1__yearly_rate_base,2022,15077
-out_ferc1__yearly_rate_base,2023,15385
-out_ferc1__yearly_rate_base,2024,15196
```

Comment:
Because these row counts vary a little bit randomly, we removed the test from the table. But forgot to remove the actual row counts.

We should add a test that the overall row counts are approximately what we expect them to be. This test is available from dbt_expectations I think.